web/requirements.txt

```diff
@@ -1,9 +1,15 @@
 cryptography==2.9.2
 Flask==1.1.2
 Flask-Bcrypt==0.7.1
+marshmallow==3.5.1
 flask-marshmallow==0.12.0
 Flask-Migrate==2.5.3
 Flask-SQLAlchemy==2.4.1
 marshmallow-sqlalchemy==0.22.3
 PyMySQL==0.9.3
+<<<<<<< HEAD
+Flask-User==1.0.2.2
+email-validator==1.1.0
+=======
 yapf==0.30.0
+>>>>>>> origin
```

Comment:
Please fix the git merge conflict comments. This breaks the file if you leave it.

Reply:
got it